packages/core/src/Cardano/types/DelegationsAndRewards.ts

```diff
@@ -31,3 +31,16 @@ export interface RewardAccountInfo {
   rewardBalance: Lovelace;
   // Maybe add rewardsHistory for each reward account too
 }
+
+export interface Cip17Pool {
+  id: PoolIdHex;
+  weight: number;
+  name?: string;
+  ticker?: string;
+}
+export interface Cip17DelegationPortfolio {
+  name: string;
+  pools: Cip17Pool[];
+  description?: string;
+  author?: string;
```

Comment:
What's this field? It's not in the CIP-17 spec 🤔

Reply:
It is in the [cip-17 schema](https://cips.cardano.org/cips/cip17/CIP-0017.json)

Reply:
Oh, I looked at the example as my canonical source by mistake, instead of the schema 🙃